proto/orc_proto.proto

```diff
@@ -142,12 +149,35 @@ message ColumnEncoding {
   optional uint32 bloomEncoding = 3;
 }
 
+message StripeEncryptionVariant {
+  repeated Stream streams = 1;
+  repeated ColumnEncoding encoding = 2;
+}
+
+// each stripe looks like:
+//   index streams
+//     unencrypted
+//     variant 1..N
+//   data streams
+//     unencrypted
+//     variant 1..N
+//   footer
+
 message StripeFooter {
   repeated Stream streams = 1;
   repeated ColumnEncoding columns = 2;
   optional string writerTimezone = 3;
+  // one for each column encryption variant
+  repeated StripeEncryptionVariant encryption = 4;
 }
 
+// the file tail looks like:
+//   encrypted stripe statistics: ColumnStripeStatistics (order by variant)
+//   stripe statistics: Metadata
+//   footer: Footer
```

Comment:
The encrypted file statistics are serialized directly into the EncryptionVariant.fileStatistics.

Reply:
My thought was that stripe statistics can get large, but file statistics don't. So rather than go through another level of stream lookup and reading, we could just inline it.

Reply:
> The encrypted file statistics are serialized directly into the EncryptionVariant.fileStatistics.
When will EncryptionVarian.fileStatistics be not empty？ Or where is the encryption variant for file statistics stored？